flink-core/src/main/java/org/apache/flink/core/fs/SafetyNetCloseableRegistry.java

```diff
@@ -186,6 +212,15 @@ private CloseableReaperThread() {
 			this.running = true;
 		}
 
+		@VisibleForTesting
+		CloseableReaperThread(String name) {
```

Comment:
Why adding a new constructor here?
If you are blocked by the existing private constructor, you can just change it to protected.

Reply:
OK, I will change it to protected.